code/modules/industrial_lift/industrial_lift.dm

```diff
@@ -424,7 +424,10 @@ GLOBAL_LIST_EMPTY(lifts)
 				var/datum/callback/land_slam = new(collided, TYPE_PROC_REF(/mob/living/, tram_slam_land))
 				collided.throw_at(throw_target, 200 * collision_lethality, 4 * collision_lethality, callback = land_slam)
 
-				SEND_SIGNAL(src, COMSIG_TRAM_COLLISION, collided)
+				//increment the hit counter signs
+				if(ismob(collided) && collided.client)
+					SSpersistence.tram_hits_this_round++
+					SEND_SIGNAL(src, COMSIG_TRAM_COLLISION, SSpersistence.tram_hits_this_round)
```

Comment:
you're changing signal behaviour here. COMSIG_TRAM_COLLISION before was not expecting a mob with a client, but you're changing it now so that it does. This is going to lead to bugs with pre-existing signal handlers.
You need to move this send signal outside of the proc and also fix whatever is expecting a mob to filter only for mobs with client instead of placing your if statement here.

Reply:
I made this signal, and my object is the only one using it, there are no pre-existing handlers. Client is checked so people can't run the counter by throwing monkeys etc on the track.